### ISSUE TITLE
Branch loses prototype status after renaming

### ISSUE THREAD

**Issue by sergej-koscejev** [git-town/git-town#3864]
### Steps to reproduce

```
$ git town hack feature/x
$ git town rename-branch feature/x feature/x-1
$ git config git-town.prototype-branches
 feature/x
```

### Command output

```shell
No failing output but the branch will get pushed on next sync.
```

**Comment by kevgo** [comment]
Yes that is totally a missing feature. Thanks for reporting!

**Comment by kevgo** [comment]
This is now released in [v15.2](https://github.com/git-town/git-town/releases/tag/v15.2.0). Check it out 😃
